### ISSUE TITLE
Texture Error

### ISSUE THREAD

**Issue by YoungGT** [primetoxinz/StacksOnStacks#22]
(I'm not good at English,so if I have any mistakes in grammar,please forgive me.Thank you.)
I had a 1.7.10 modpack with this mod and Automagy.And when I held Automagy's redcrystal to right-click blocks,there was a mistake of texture  of storing crystal.It should be like [this](http://image.jiantuku.com/17-9-9/51903570.jpg?attname=file_1504966538370_e4fd.png&e=1504966810&token=el7kgPgYzpJoB23jrChWJ2gV3HpRl0VCzFn8rKKv:p9zGPrTwef5_E4LWFVCquYA4PG4=),but it didn't,like [this](http://image.jiantuku.com/17-9-9/11767098.jpg?attname=file_1504966538247_5333.png&e=1504966810&token=el7kgPgYzpJoB23jrChWJ2gV3HpRl0VCzFn8rKKv:MBjWKXpzmdqyvLAJ4KEbehc0Meo=).

**Comment by primetoxinz** [comment]
Yeah this was a know issue back in the day, but the 1.7.10 was lost in harddrive failure so no bugs can be fixed.
